templates/corporate/zephyr-mirror.html

```diff
@@ -1,5 +1,9 @@
 {% extends "zerver/portico.html" %}
 
+{% block title %}
+<title>Zulip zephyr: Mirroring information</title>
```

Comment:
Did not mark this as a translated string. Also wasn't sure if this is a maintained page?

Reply:
Should be “Zulip: Zephyr mirroring information” ([Zephyr](https://en.wikipedia.org/wiki/Zephyr_(protocol)) is a proper noun and is what’s being mirrored).

Reply:
Then we should also update `zephyr.html` because that's what I used as a reference for making this title.

Neither template has the page content marked for translation, so I'm going to keep the title element also English only.